src/autorest-core/lib/pipeline/commonmark-documentation.ts

```diff
@@ -61,7 +61,7 @@ export async function ProcessCodeModel(codeModel: DataHandleRead, scope: DataSto
       const nodeOriginal = CloneAst(node);
       const key = nodeOriginal.key.value;
       const origKey = key + "_Original";
-      nodeOriginal.key.value = origKey
+      nodeOriginal.key.value = origKey;
```

Comment:
This gives me an idea, do you think we could add a linting task to our gulp build which sanitizes typescript
Something like [this](https://github.com/palantir/tslint) (ofc we need to see if it really is effective)